app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user &. authenticate(params[:session][:password])
```

Comment:
dùng &. thì ko viết tách ra như vậy đâu
```suggestion
    if user&.authenticate(params[:session][:password])
```